iq-apply-conf.py

```diff
@@ -68,6 +68,9 @@ def main():
     # Admin level configuration and integrations
     nexus_administration(config)
 
+    # ROOT level configuration
+    root_configuration(config)
```

Comment:
root_configuration is deprecated code. Please update your baseline to the latest checkin on master branch.